uint32 sat
uint32 week
float64 ura
uint32 health
float64 tgd1
float64 tgd2
uint32 AODC
uint32 toc
float64 a0
float64 a1
float64 a2
uint32 AODE
uint32 toe
float64 RootA
float64 ecc
float64 omg
float64 delta_N
float64 M0
float64 OMG_0
float64 OMG
float64 i0
float64 I_DOT
float64 cuc
float64 cus
float64 crc
float64 crs
float64 cic
float64 cis

